src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -64,6 +65,9 @@ export class EditConcept extends Component {
     selectedAnswers: PropTypes.array.isRequired,
     removeAnswer: PropTypes.func.isRequired,
     createNewAnswerRow: PropTypes.func.isRequired,
+    deleteConcept: PropTypes.func.isRequired,
```

Comment:
How is the above line related to this ticket?

Reply:
A reference to a concept cannot be edited directly, therefore, it has to first be deleted from the Dictionary, then Recreated. This prop is being called within the method: `updateConceptReference()`